samples/enums/underlying.jakt

```diff
@@ -0,0 +1,13 @@
+enum CuteLittleEnum: u64 {
+    A = 1
+    B
+    C = 69
```

Comment:
What about a test for an initial enum variant without an explicit value? That should be 0, right, how about we check that?